polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaInterpreter.scala

```diff
@@ -58,31 +59,6 @@ class ScalaInterpreter(
 
   protected val previousSources: mutable.HashMap[String, ScalaSource[this.type]] = new mutable.HashMap()
 
-  protected val importToRuntime: Importer[runtime.universe.type, global.type] =
-    runtime.universe.internal.createImporter(global)
-
-  protected val importFromRuntime: Importer[global.type, runtime.universe.type] =
-    global.internal.createImporter(scala.reflect.runtime.universe)
-
-  protected val runtimeMirror: runtime.universe.Mirror =
-    scala.reflect.runtime.universe.runtimeMirror(symbolTable.classLoader)
-
-  // the ToolBox is only used to eta-expand methods into runtime functions via eval()
-  // it's not exactly the cleanest way to do that, but it deals with 22 arities for us
-  protected object toolsFrontEnd extends scala.tools.reflect.FrontEnd {
-    var currentOutputs: Queue[IO, Result] = _
-
-    def display(info: Info): Unit = if (currentOutputs != null) {
-      currentOutputs.enqueue1(CompileErrors(
-        KernelReport(new Pos(info.pos), info.msg, info.severity.id) :: Nil
-      )).unsafeRunAsyncAndForget()
-    }
-
-    def interactive(): Unit = ()
-  }
```

Comment:
Did this get moved anywhere or do we no longer need the special display stuff?

Reply:
I realized nothing was being done with this (i.e. I never set `currentOutputs`, so this wasn't doing anything).

Reply:
I think the original idea was to capture reflection errors as compile-time errors rather than runtime errors. It was probably because I was debugging some problem with reflecting methods into functions. But apparently it's not used anymore, so it goes away.

Reply:
gotcha. catching reflection errors at compile time sounds cool though :)